src/it/resources/com/google/checkstyle/test/chapter5naming/rule523methodnames/InputMethodName.java

```diff
@@ -116,7 +173,7 @@ void testingFoo2() {}
   void testingFoo_foo2() {}
 
   @RepeatedTest(2)
-  void testing_01232() {}
+  void testing_01232() {} // violation 'Method name 'testing_01232' must match pattern'
```

Comment:
Sorry, this will revert back to being disallowed due to [this](https://github.com/checkstyle/checkstyle/pull/15617#discussion_r1752998366) feedback